### PR TITLE
add Icon, social graphic and update brand colors for steampipe hub

### PR DESCRIPTION
**Pull request by dboeke**
Updated the docs index and added social graphics/icon for use in steampipe hub

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 organization: francois2metz
 category: ["public cloud"]
-brand_color: "#FF9900"
+brand_color: "#45AFE4"
 display_name: "Scalingo"
 short_name: "scalingo"
 description: "Steampipe plugin for querying apps, addons and more from Scalingo."
 og_description: "Query Scalingo with SQL! Open source CLI. No DB required."
+og_image: "/images/plugins/francois2metz/scalingo-social-graphic.png"
+icon_url: "/images/plugins/francois2metz/code.svg"
```
Comment (francois2metz) [inline docs/index.md:10]: This shouldn't be `scalingo.svg` instead ?

**Comment by francois2metz** [comment]
@dboeke Awesome, thanks for your work and the future inclusion on the hub!
There is one question on the `icon_url`, as the filename doesn't match the svg you uploaded.

**Comment by e-gineer** [comment]
Yes, it should be `scalingo.svg` not `code.svg` ... FYI that these changes are currently in the development branch of the hub and will be available soon - https://github.com/turbotio/hub.steampipe.io/tree/development/public/images/plugins/francois2metz

**Comment by francois2metz** [comment]
Thanks for the update. I merged the PR manually.

**Pull request closed without merging**